requirements/test.in

```diff
@@ -11,3 +11,4 @@ pytest-cov
 pytest-mock
 tox
 virtualenv
+pydocstyle~=3.0
```

Comment:
TIL `~=`

I'm guessing this is due to https://gitlab.com/pycqa/flake8-docstrings/issues/36 and https://github.com/PyCQA/pydocstyle/issues/375 ? Should we add a comment to unpin when those are resolved?